rfcs/0027-formalize-documentation-organization.md

```diff
@@ -0,0 +1,430 @@
+how to successfully install, use, and develop for TVM. The documenation project grew with
+the community, to address the immediate needs of the developer community. However, one
+consistent piece of feedback is that the documentation is difficult to navigate, with beginner
+material mixed in with advanced material. As a result, it can be difficult for new users
+to find the exact information they need, and can work against the vision of the project.
+
+This RFC aims to refactor the organization of the TVM docs, loosely following the [formal
+documentation style described by Divio](https://documentation.divio.com). This system has been chosen
+because it is a:
+
+> "simple, comprehensive and nearly universally-applicable scheme. It is proven
+> in practice across a wide variety of fields and applications."
+
+This RFC is primarily concerned with the organization of the documents, and not
+the content. As such, the implementation of this RFC would move documents, and
+only create new documents as top-level placeholders, indexes, and documentation
+about the system itself.
+
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## The Four Document Types
+
+### Introductory Tutorials
+
+These are step by step guides to introduce new users to a project. A successful
+introductory tutorial successfully get the user engaged with the software
+without necessarily explaining why the software works the way it does. Those
+explanations can be saved for other document types; the introductory tutorial
+focuses on a successful first experience. These are the most important docs to
+turning newcomers into new users and developers. A fully end-to-end tutorial,
+from installing TVM and supporting ML software, to creating and training a
+model, to compiling to different architectures will give a new user the
+opportunity to use TVM in the most efficient way possible.
+
+Tutorials need to be repeatable and reliable, because the lack of success means
+a user will look for other solutions.
+
+### How-to Guides
+
+These are step by step guides on how to solve particular problems. The user can
+ask meaningful questions, and the documents provide answers. An examples of
+this type of document might be, “how do I compile an optimized model for ARM
+architecture?” or “how do I compile and optimize a TensorFlow model?” These
+documents should be open enough that a user could see how to apply it to a new
+use case. Practical usability is more important than completeness. The title
+should tell the user what problem the how-to is solving.
+
+How are tutorials different from how-tos? A tutorial is oriented towards the
+new developer, and focuses on successfully introducing them to the software and
+community. A how-to in contrast focuses on accomplishing a specific task within
+the context of basic understanding. A tutorial helps to onboard, a how-to helps
```

Comment:
sorry here i see this now. it seems like the distinction is scope--the tutorial being broader basic understanding and the how-to being focused on a singular task. can you reflect that in the Tutorial description above? I want us to be as clear as possible here.

Reply:
The primary difference is a how-to requires basic prior knowledge of the software, and answers a question that user would have. A tutorial assumes no knowledge, and is meant show a user what they can accomplish with the software.